src/components/nve-switch/nve-switch.component.ts

```diff
@@ -0,0 +1,148 @@
+import { customElement, property, query, state } from 'lit/decorators.js';
+import { INveComponent } from '@interfaces/NveComponent.interface';
+import { CSSResultArray, html, LitElement, TemplateResult } from 'lit';
+import { watch } from '../../utils/watch';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { live } from 'lit/directives/live.js';
+import { classMap } from 'lit/directives/class-map.js';
+import styles from './nve-switch.styles';
+/**
+ * En toggle-button med støtte for ikoner.
```

Comment:
Går det an å skrive vippe-bryter i stedet for toggle-button? Forstår folk det? Eller finnes det noe bedre norsk begrep? av/på-knapp?

Reply:
Skrevet om til vippe-bryter